research/picai/README.md

```diff
@@ -16,17 +16,7 @@ pip install --upgrade pip poetry
 poetry install --with "picai"
 ```
 
-This will setup an environment with the proper dependencies to run the provided scripts out of the box.
-
-If you wish to also be able to reproduce the pre-processing described in subsequent sections, you can modify the aforementioned series of commands as follows:
-```bash
-virtualenv "ENV_PATH"
-source "ENV_PATH/bin/activate"
-pip install --upgrade pip poetry
-poetry install --with "picai, picai-preprocessing"
```

Comment:
It might be worth at least noting that to setup the environment with the necessary library components to do picai experiments, you want to add "picai" to the with statement in our main documentation page?

Reply:
One other super minor thing is that our env setup instructions have moved to `CONTRIBUTING.MD`

Reply:
In regards to the first comment, that is still included in the README.md just above what I deleted. I will update the location of the document to CONTRIBUTING.md